_includes/releases/v22.2/v22.2.7.md

```diff
@@ -0,0 +1,147 @@
+- Since [v22.1](../releases/v22.1.html) when [rangefeed enablement](../v22.2/create-and-configure-changefeeds.html#enable-rangefeeds) overrides in span configs were introduced, rangefeed requests that reached spans outside the range would not cause range cache invalidation due to the setting being checked first, thus requests could repeatedly hit the same incorrect range, causing errors until cache invalidation or node restart. This fix correctly checks that the span is within the range prior to checking the enablement settings, thus invalidating the cache when a request reaches an incorrect range and causes subsequent requests to successfully reach the correct range. [#97659][#97659]
+- Previously the [declarative schema changer](../v22.2/online-schema-changes.html#declarative-schema-changer) would emit alarming messages of the form: `failed building declarative schema change targets for...`. These were non-severe in nature and are now disabled by default. [#98258][#98258]
+- Fixed a bug where new schema changes which used the [declarative schema changer](../v22.2/online-schema-changes.html#declarative-schema-changer) in a mixed version state upgrading from [v22.1](../releases/v22.1.html) did not execute properly. The impacted [schema changes](../v22.2/online-schema-changes.html) are `ADD COLUMN`, `DROP COLUMN`, `ALTER PRIMARY KEY`, `CREATE INDEX`, `DROP OWNED BY`, `COMMENT ON`, and `DROP INDEX`. [#98379][#98379]
+- Fixed a bug where if an [`UPDATE`](../v22.2/update.html) was performed during an ongoing [`ADD COLUMN`](../v22.2/alter-table.html#add-column) or [`DROP COLUMN`](../v22.2/alter-table.html#drop-column) on a table, the update could incorrectly fail due to a duplicate key error. [#98505][#98505]
+- Fixed a bug where CockroachDB could encounter the internal error `concurrent txn use detected`. The bug was introduced in [v22.2.0](../releases/v22.2.html). [#98406][#98406]
+- Fixed a bug that could crash the process when a query contained a literal [tuple expression](../v22.2/scalar-expressions.html#tuple-constructors) with more than two elements and only a single label, e.g., `((1, 2, 3) AS foo)`. [#98315][#98315]
+- Fixed a bug where the stats columns on the [Transaction Fingerprint overview page](../v22.2/ui-transactions-page.html#transaction-fingerprints-view) did not not continuously increment. [#98336][#98336]
+- Fixed a bug in evaluation of `ANY`, `SOME`, and `ALL` [sub-operators](../v22.2/functions-and-operators.html#operators) that would cause expressions like `NULL = ANY(ARRAY[]::INT[])` to return `NULL` instead of `false`. [#98165][#98165]
+- Allow users with `VIEWACTIVITY`/`VIEWACTIVITYREDACTED` [permissions](../v22.2/security-reference/authorization.html#role-options) to access the [`crdb_internal.ranges_no_leases`](../v22.2/crdb-internal.html) table, necessary to view important [DB Console](../v22.2/ui-overview.html) pages (such as the Databases page, including database details, and database tables). [#98645][#98645]
```

Comment:
Nit: Fix tense here, suggest:

Users with `VIEWACTIVITY`/`VIEWACTIVITYREDACTED` [permissions](../v22.2/security-reference/authorization.html#role-options) are now allowed to access the...